### PR TITLE
Get files with open modifications

### PR DESCRIPTION
**Pull request by ableuler**

### CODE REVIEW

File: src/gitlab/client.js [hunk 1]
```diff
@@ -368,6 +368,60 @@ class GitlabClient {
     })
 
   }
+
+  getMergeRequests(projectId, queryParams={}) {
+    let headers = this.getBasicHeaders();
+    const url = projectId ? `${this._baseUrl}projects/${projectId}/merge_requests` :
+      `${this._baseUrl}/merge_requests`
```
Comment (ciyer) [inline src/gitlab/client.js]: There is a "/" after ${this._baseUrl} in one case, but none in the other. This should be standardized. (I prefer always having a "/"...)

File: src/gitlab/client.js [hunk 2]
```diff
@@ -368,6 +368,60 @@ class GitlabClient {
...
+        // For each MR get the changes introduced by the MR, creates an array
+        // of promises.
+        const mergeRequestsChanges = mergeRequests.map((mergeRequest) => {
+          return this.getMergeRequestChanges(projectId, mergeRequest.iid)
+        });
+
+        // On resolution of all promises, form an object which lists for each file
+        // the merge requests that modify this file.
+        return Promise.all(mergeRequestsChanges)
+          .then((MRchanges) => {
```
Comment (ciyer) [inline src/gitlab/client.js]: Unusual to use an initial capital letter for a method parameter. Let's call this "mrChanges".

File: src/gitlab/client.js [hunk 3]
```diff
@@ -368,6 +368,60 @@ class GitlabClient {
...
+            const openMrs = {};
+            MRchanges.forEach((MRchange) => {
```
Comment (ciyer) [inline src/gitlab/client.js]: See above re: param names. Rename to "mrChange".

File: src/gitlab/client.js [hunk 4]
```diff
@@ -368,6 +368,60 @@ class GitlabClient {
...
+              const changesArray = MRchange.changes;
+              const MRIid = MRchange.iid;
```
Comment (ciyer) [inline src/gitlab/client.js]: See above re: param names. Same applies to locals. Use "mrIid".

File: src/gitlab/client.js [hunk 5]
```diff
@@ -368,6 +368,60 @@ class GitlabClient {
...
+              changesArray
+                .filter((change) => change.old_path === change.new_path)
+                .forEach((change) => {
+                  if (!openMrs[change.old_path]) openMrs[change.old_path] = [];
+                  openMrs[change.old_path] = openMrs[change.old_path].concat(MRIid)
```
Comment (ciyer) [inline src/gitlab/client.js]: Is there a reason to use concat and not push? E.g., 
```
openMrs[change.old_path].push(mrIid)
```

**Comment by ableuler** [comment]
@ciyer thanks for the suggestions, all requested changes are implemented.

**Comment by ableuler** [comment]
Closes #236

**Pull request merged**